.github/workflows/docs.yml

```diff
@@ -1,10 +1,13 @@
 name: Docs
 
 on:
+  workflow_dispatch:
   pull_request: ~
   push:
     branches:
       - main
+  schedule:
+    - cron: '0 7 * * *'
```

Comment:
I'd really would like this to reside in a dedicated commit for git log historic reasons. This change to run it nightly is otherwise completely hidden.

Reply:
Sure. I've diverted this to #119.

Reply:
👍 thx! btw. a dedicated commit inside this PR would have been also fine ;)